rasa_core/policies/two_stage_fallback.py

```diff
@@ -146,7 +159,9 @@ def persist(self, path: Text) -> None:
         meta = {
             "nlu_threshold": self.nlu_threshold,
             "core_threshold": self.core_threshold,
-            "fallback_action_name": self.fallback_action_name,
+            "fallback_core_action_name": self.fallback_action_name,
+            "fallback_nlu_action_name": self.fallback_nlu_action_name,
+            "deny_suggestion_intent_name": self.deny_suggestion_intent_name,
```

Comment:
this will break existing models, please increase the model compatibility version